recipe/meta.yaml

```diff
@@ -27,10 +26,13 @@ requirements:
     - python
     - pip
     - distribute
+    - openblas        0.2.20   # [unix]
+    - mkl             2018.0.3  # [win]
```

Comment:
Please read [this doc about `blas`]( https://conda-forge.org/docs/blas.html ).

Also pinnings are handled at a global level. So no need to pin here.

We currently haven't figured out MKL support due to legal gray areas, but are looking into it. Though there is `openblas` on Windows.

Reply:
@jakirkham I've read [the `blas` link you sent](https://conda-forge.org/docs/blas.html). Was it sent on a google translate round-robin, by any chance? 

The situation with this feedstock is this:
- PySparse is compatible with Python 2.x only
- Python 2.7 is apparently only available with `vc9` on `[win]`
- `openblas` is apparently only available with `vc14` on `[win]`
- I pinned `openblas` on `[unix]` at 0.2.20 because my feedstock ceased to build when `openblas` 0.3.3 went up
- as a result, on the recommendation of @isuruf, I required `mkl` on `[win]`
- PySparse' `setup.py` [couldn't find `mkl`](https://ci.appveyor.com/project/conda-forge/pysparse-feedstock/builds/19419276), even though it could [when this was a `staged-recipe`](https://ci.appveyor.com/project/conda-forge/staged-recipes/build/1.0.27812)
- as a result, I'm trying to pin `mkl` to 2018.0.3, which is what worked ast a `staged-recipe`
- AppVeyor sits on every one of my changes for hours, making troubleshooting this virtually impossible
- I've spent months banging my head against these feedstocks and I really hate conda-forge right now